index.d.ts

```diff
@@ -166,7 +166,10 @@ export interface OidcClientSettings {
   MetadataServiceCtor?: MetadataServiceCtor;
   /** An object containing additional query string parameters to be including in the authorization request */
   extraQueryParams?: Record<string, any>;
+  clockService?: ClockService;
+}
 
+export interface ClockService {
```

Comment:
This is already defined an exported on [line 317](https://github.com/IdentityModel/oidc-client-js/blob/e39b69d89a6d42e4429c3e5561960316596b746b/index.d.ts#L317).